resources/js/autocomplete.js

```diff
@@ -77,8 +77,10 @@ export const autocomplete = (config) => ({
     },
 
     select(replacement) {
+        replacement ??= this.getReplacementFromSelectedResult() ?? this.activeToken.word;
```

Comment:
This code in english: the `replacement` is either the specified value from the selected autocomplete option (when present) OR a fallback of the current word the user is typing (thus, hitting enter just closes the autocomplete panel and inserts a space so the user can keep typing).